learn.html

```diff
@@ -114,13 +114,13 @@
         <div class="col-xs-12 col-sm-9 content_stuff">
             <section id="Data-From-Subsystem" class="group">
                 <h1>Work with data from a subsystem</h1>
-                    <p>Data from the Net Zero house includes measurements from various instruments around the house in one minute increments. Each variable, also called channels in  NIST documentation, provides the readings from one instrument, and the instruments are grouped into subsystems like HVAC, Electrical, DHW. </p>
+                    <p>Data from the Net-Zero house includes measurements from various instruments around the house in one minute increments. Each variable, also called channels in  NIST documentation, provides the readings from one instrument, and the instruments are grouped into subsystems like HVAC, Electrical, DHW. </p>
                     <p> This tutorial walks through loading data from a particular subsystem.</p>
  
         <div id="Data-From-Subsystem-A" class="subgroup">
 
                 <h3>Get the data you need</h3>
-                <p>If you want to analyze net-zero data for specific subsystems, rather than data for the whole house, you can download the data from the Net Zero website and import it into R, or you can use the download link directly.</p>
+                <p>If you want to analyze net-zero data for specific subsystems, rather than data for the whole house, you can download the data from the Net-Zero website and import it into R, or you can use the download link directly.</p>
```

Comment:
@maabrown Missed one of the `net-zero` to `Net-Zero` fixes.  `If you want to analyze net-zero data for specific subsystems`